src/services/services.ts

```diff
@@ -2912,6 +2919,63 @@ module ts {
 
                 return filteredMembers;
             }
+
+            function isExternalModuleName(token: Node): boolean {
+                if (token && token.parent && token.kind === SyntaxKind.StringLiteral && isInStringOrRegularExpressionOrTemplateLiteral(token)) {

```

Comment:
When `token` is a terminated string literal but the cursor position resides beyond the `end` of the token.